lib/siblings/lookup.ex

```diff
@@ -142,10 +142,14 @@ defmodule Siblings.Lookup do
 
     opts = Map.get(worker_spec, :options, [])
     {shutdown, opts} = Keyword.pop(opts, :shutdown, 5_000)
+    {throttler, opts} = Keyword.pop(opts, :throttler, [])
+
+    throttler = for thr <- throttler, into: %{}, do: {{worker, thr}, name}
```

Comment:
Store `Throttler` config. It is specified as a list of states when it should be used.